server/helpers/generate-token.js

```diff
@@ -0,0 +1,10 @@
+import jwt from 'jsonwebtoken';
+import env from 'dotenv'
```

Comment:
you don't need to import `dotenv` here. it's done only once, most of the time in the entry file